spec/factories/task_files.rb

```diff
@@ -12,6 +12,13 @@
       fileable { build(:test) }
     end
 
+    trait :exportable do
```

Comment:
Why is this trait called "exportable"? It somewhat implies to me that others can't be exported which I don't see at the moment.

Reply:
If the attributes in this trait aren't set, the exporter sets default values, because the fields are required, except internal-description where nil would turn to an empty string and therefore isn't equal as well.
It's basically just a helper to prepare the files in a way that i can export them, then reimport them and everything should be the same.
Maybe we can call it `:reimportable`? It feels a bit clunky though

Reply:
Okay, this makes sense. By providing fixed default values, you can expect those within your tests.
I don't have a strong opinion on the name, choose whatever you prefer. I see pros and cons for both names and would maybe leave it as it is for now.